### ISSUE TITLE
Mouse CursorModeCapture broken in WASM

### ISSUE THREAD

**Issue by harbdog** [hajimehoshi/ebiten#2634]
### Ebitengine Version

2.5

### Operating System

- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] FreeBSD
- [ ] OpenBSD
- [ ] Android
- [ ] iOS
- [ ] Nintendo Switch
- [ ] Xbox
- [X] Web Browsers

### Go Version (`go version`)

1.19.6

### What steps will reproduce the problem?

Created an example and gh-pages to host the wasm example in my fork:

https://harbdog.github.io/ebiten/

Branch: `wasm_mouse_capture`
https://github.com/harbdog/ebiten/tree/wasm_mouse_capture

```bash
git checkout wasm_mouse_capture
wasmserve.exe ./examples/capture
```

It works when not using WASM:
```bash
git checkout wasm_mouse_capture
go run ./examples/capture
```

### What is the expected result?

Mouse capture should work in browser the same as it does in desktop application mode.

### What happens instead?

The mouse tends to move only in a single direction or jumps around unexpectedly.

**Comment by harbdog** [comment]
It seems that using the Escape key to toggle capture may conflict a bit with the browser also using Escape to release capture, that might be the source of some of the weird behavior.

However I also tried changing to use Space to toggle capture, which did at least address some of the weirdness with re-capturing the cursor. However, what I noticed was that it still showed only the X mouse movement to register when using Chrome.

However in Edge, it only has Y mouse movement registering.

In Firefox it registers both X and Y mouse movement, but the movement is not smooth, almost "drunk" like.
